### PR TITLE
XW-4806 | AB Test of RV

### PR DESCRIPTION
**Pull request by bkoval**
@Wikia/x-wing

### CODE REVIEW

**Review by rybmat — APPROVED** [review]

File: app/components/recommended-video.js [hunk 1]
```diff
@@ -26,8 +27,14 @@ export default Component.extend(NoScrollMixin, {
 				action: trackActions.impression,
 			});
 
-			// Uncomment after XW-4771 test is done
-			// this.initRecommendedVideo();
+			const isClickToPlay = inGroup('RECOMMENDED_VIDEO_AB', 'CLICK_TO_PLAY');
```
Comment (latata) [inline app/components/recommended-video.js]: as we load AbTesting module asynchronously are we sure it will be always available here?

Comment (latata) [reply app/components/recommended-video.js]: Probably yes as we wait 3s but who knows... :P

File: app/components/recommended-video.js [hunk 2]
```diff
@@ -19,16 +20,28 @@ export default Component.extend(NoScrollMixin, {
 	init() {
 		this._super(...arguments);
 
-		run.later(() => {
-			track({
-				category: 'related-video-module',
-				label: 'reveal-point',
-				action: trackActions.impression,
-			});
+		window.waitForInstantGlobals(function () {
```
Comment (xkxd) [inline app/components/recommended-video.js]: they are not so instant if you have to wait for them :trollface:

Comment (bkoval) [reply app/components/recommended-video.js]: 🥇

**Review by latata — APPROVED** [review]

File: app/components/recommended-video.js [hunk 3]
```diff
@@ -170,7 +170,10 @@ export default Component.extend(NoScrollMixin, {
 	},
 
 	getVideoData() {
-		return fetch(`https://cdn.jwplayer.com/v2/playlists/${this.get('playlistId')}?related_media_id=${this.get('relatedMediaId')}`).then((response) => response.json());
+		/* eslint-disable-next-line max-len */
+		const url = `https://cdn.jwplayer.com/v2/playlists/${this.get('playlistId')}?related_media_id=${this.get('relatedMediaId')}`;
```
Comment (hakubo) [inline app/components/recommended-video.js]: new lines are ommited in fetch from what I tested today
so you can safely break the line!

**Comment by rybmat** [comment]
please DO NOT CLICK "update branch" until https://github.com/Wikia/mobile-wiki/pull/656 is merged

**Pull request merged**